graph/db/sql_store.go

```diff
@@ -3485,27 +3486,30 @@ func buildNodeWithBatchData(dbNode sqlc.GraphNode,
 	var pub [33]byte
 	copy(pub[:], dbNode.PubKey)
 
-	node := &models.Node{
-		PubKeyBytes: pub,
-		Features:    lnwire.EmptyFeatureVector(),
-		LastUpdate:  time.Unix(0, 0),
-	}
+	node := models.NewV1ShellNode(pub)
 
 	if len(dbNode.Signature) == 0 {
 		return node, nil
 	}
 
 	node.AuthSigBytes = dbNode.Signature
-	node.Alias = dbNode.Alias.String
-	node.LastUpdate = time.Unix(dbNode.LastUpdate.Int64, 0)
+
+	if dbNode.Alias.Valid {
+		node.Alias = fn.Some(dbNode.Alias.String)
+	}
+	if dbNode.LastUpdate.Valid {
+		node.LastUpdate = time.Unix(dbNode.LastUpdate.Int64, 0)
+	}
```

Comment:
is there a reason why `node.LastUpdate` is non-optional when the db field is also nullable, whereas alias is optional for example?

Reply:
oh yes, it was also cause things like alias & color do become actual optional fields but update time must always be set & isnt really optional in that sense. 

but yeah, we can make this more explicit later with helpers like "ExtractV1Node" or something